common/utils/utility.py

```diff
@@ -157,6 +157,7 @@ def decrypt_data(encryption_key, iv, data):
     decrypt_result = crypto.SKENC_DecryptMessage(encryption_key, crypto.hex_to_byte_array(iv), data_byte)
     result = base64.b64decode(crypto.byte_array_to_base64(decrypt_result))
     logger.info("Decryption Result at Client - %s ", result)
+    return result.decode("UTF-8")
```

Comment:
It make sense if it returns a value. Other wise just for printing de-crypted message we are calling this function. I will add the return statement in function description. 

Line 157 will decrypt the data_byte using encryption_key and iv and we will get the base64 encode string and in 158 it will decode from base64 to string.

Reply:
decrypt_result is already a byte_array. We don't need to encode it as base64 and again do base64 decoding. We can probably use crypto.byte_array_to_string(decrypt_result )

Reply:
Done